src/main/java/org/galatea/starter/domain/IexHistoricalPriceId.java

```diff
@@ -0,0 +1,33 @@
+package org.galatea.starter.domain;
+
+import java.io.Serializable;
+import java.util.Objects;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
+
+@AllArgsConstructor(access = AccessLevel.PUBLIC)
+@NoArgsConstructor(access = AccessLevel.PUBLIC)
+public class IexHistoricalPriceId implements Serializable {
+
+  private String symbol;
+  private String date;
+
+  @Override
+  public boolean equals(final Object o) {
+    if (this == o) {
+      return true;
+    }
+    if (o == null || getClass() != o.getClass()) {
+      return false;
+    }
+    IexHistoricalPriceId iexHistoricalPriceId = (IexHistoricalPriceId) o;
+    return symbol.equals(iexHistoricalPriceId.symbol) && date.equals(iexHistoricalPriceId.date);
+  }
+
+  @Override
+  public int hashCode() {
+    return Objects.hash(symbol, date);
+  }
```

Comment:
Equals and HashCode functions can be automatically generated by Lombok using the `@EqualsAndHashCode` annotation, alternatively use `@Data` which includes `@EqualsAndHashCode` as well as `@ToString`, `@Getters`, `@Setters`, and `@RequiredArgsConstructor`.